src/adhocracy_core/adhocracy_core/auditing/__init__.py

```diff
@@ -92,8 +99,17 @@ def audit_resources_changes_callback(request: Request,
     registry = request.registry
     changelog_metadata = registry.changelog.values()
     user_name, user_path = _get_user_info(request)
-    for meta in changelog_metadata:
-        _log_change(user_name, user_path, meta)
+    for change in changelog_metadata:
+        if _is_audit_change(change):
+            action_name = _get_entry_name(change),
+            sheets = _get_content_sheets(change, registry)
+            sheet_data = _get_sheet_data(sheets, request)
+            log_auditevent(change.resource,
+                           action_name,
+                           user_name=user_name,
+                           user_path=user_path,
+                           sheet_data=sheet_data)
+            transaction.commit()
 
```

Comment:
| Not part of this pull request, but why do we need a commit for every single auditlog?

It was already there, I don't see why. I think we can remove it.